.travis.yml

```diff
@@ -17,6 +17,34 @@ matrix:
     env: PUPPET_GEM_VERSION="~> 4.0"
   - rvm: 2.4.1
     env: PUPPET_GEM_VERSION="~> 5.0"
+  - rvm: default
+    sudo: required
+    dist: trusty
+    services: docker
+    env: BEAKER_set="debian-8"
+    bundler_args:
+    script: sudo service docker restart ; sleep 10 && bundle exec rspec spec/acceptance/*_spec.rb
```

Comment:
Any particular reason to use this over `bundle exec rake beaker`? Also wondering why the docker restart and sleep is needed. We run docker tests in many repos without it.

Reply:
We've been running things this way for years on other modules. We had problems with docker in Travis a few years ago so we added the restart + sleep and it fixed it, but it might not be required anymore. As for `rake beaker`, I was not aware of it; maybe it didn't exist when we started doing that.

Reply:
That's very likely :)